### PR TITLE
Docs for apt and yum repos

### PR DESCRIPTION
**Pull request by alexcb**
- create docs for installing earthly via the deb and rpm repos.

Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

File: docs/alt-installation.md [hunk 1]
```diff
@@ -24,6 +24,83 @@ and then restart your shell.
 
 For instructions on how to install `earthly` for CI use, see the [CI integration guide](./ci-integration.md).
 
+### Installing from Earthly repositories
+
+Earthly can be installed for debian and redhat based linux distributions via the earthly deb and rpm repositories.
```
Comment (dchw) [inline docs/alt-installation.md:29]:
```suggestion
Earthly can be installed for Debian and RedHat based Linux distributions via the Earthly deb and rpm repositories.
```

File: docs/alt-installation.md [hunk 2]
```diff
@@ -24,6 +24,83 @@ and then restart your shell.
...
+
+All of our binaries are signed with our [PGP key](https://pkg.earthly.dev/earthly.pgp); which has the fingerprint:
+
+    5816 B221 3DD1 CEB6 1FC9 52BA B118 5ECA 33F8 EB64
```
Comment (dchw) [inline docs/alt-installation.md:33]:
```suggestion
    `5816 B221 3DD1 CEB6 1FC9 52BA B118 5ECA 33F8 EB64`
```

File: docs/alt-installation.md [hunk 3]
```diff
@@ -24,6 +24,83 @@ and then restart your shell.
...
+
+#### Debian-based repositories (including Ubuntu)
+
+Debian-based linux users (e.g. debian, ubuntu, mint, etc) can use our apt repo to install earthly.
```
Comment (dchw) [inline docs/alt-installation.md:37]:
```suggestion
Debian-based Linux users (e.g. Debian, Ubuntu, Mint, etc) can use our apt repo to install Earthly.
```

File: docs/alt-installation.md [hunk 4]
```diff
@@ -24,6 +24,83 @@ and then restart your shell.
...
+
+Before installing earthly, you must first setup the earthly apt repo.
```
Comment (dchw) [inline docs/alt-installation.md:39]:
```suggestion
Before installing Earthly, you must first setup the Earthly apt repo.
```

File: docs/alt-installation.md [hunk 5]
```diff
@@ -24,6 +24,83 @@ and then restart your shell.
...
+
+    sudo apt-get update
+    sudo apt-get install \
+       apt-transport-https \
+       ca-certificates \
+       curl \
+       gnupg \
+       lsb-release
+
+2. Download earthly's GPG key:
```
Comment (dchw) [inline docs/alt-installation.md:53]:
```suggestion
2. Download Earthly's GPG key:
```

File: docs/alt-installation.md [hunk 6]
```diff
@@ -24,6 +24,83 @@ and then restart your shell.
...
+
+    curl -fsSL https://pkg.earthly.dev/earthly.pgp | sudo gpg --dearmor -o /usr/share/keyrings/earthly-archive-keyring.gpg
+
+3. Setup the stable repo:
+
+    echo \
+      "deb [arch=amd64 signed-by=/usr/share/keyrings/earthly-archive-keyring.gpg] https://pkg.earthly.dev/deb \
+      stable main" | sudo tee /etc/apt/sources.list.d/earthly.list > /dev/null
+
+4. Install earthly:
```
Comment (dchw) [inline docs/alt-installation.md:67]:
```suggestion
4. Install Earthly:
```

File: docs/alt-installation.md [hunk 7]
```diff
@@ -24,6 +24,83 @@ and then restart your shell.
...
+
+    sudo apt-get update
+    sudo apt-get install earthly
+
+
+#### Fedora repositories
+
+Fedora users can use our rpm repo to install earthly.
```
Comment (dchw) [inline docs/alt-installation.md:77]:
```suggestion
Fedora users can use our rpm repo to install Earthly.
```

File: docs/alt-installation.md [hunk 8]
```diff
@@ -24,6 +24,91 @@ and then restart your shell.
...
+
+1. Install plugins required to manage DNF repositories:
+
+    sudo dnf -y install dnf-plugins-core
+
+2. Add the earthly repo to your system:
```
Comment (dchw) [inline docs/alt-installation.md:83]:
```suggestion
2. Add the Earthly repo to your system:
```

File: docs/alt-installation.md [hunk 9]
```diff
@@ -24,6 +24,91 @@ and then restart your shell.
...
+
+    sudo dnf config-manager \
+        --add-repo \
+        https://pkg.earthly.dev/earthly.repo
+
+3. Install earthly:
```
Comment (dchw) [inline docs/alt-installation.md:89]:
```suggestion
3. Install Earthly:
```

Comment (dchw) [inline docs/alt-installation.md:89]:
```suggestion
3. Install Earthly:
```

Comment (dchw) [inline docs/alt-installation.md:89]:
```suggestion
3. Install Earthly:
```

File: docs/alt-installation.md [hunk 10]
```diff
@@ -24,6 +24,91 @@ and then restart your shell.
...
+
+    sudo dnf install earthly
+
+#### Centos repositories
```
Comment (dchw) [inline docs/alt-installation.md:93]:
```suggestion
#### CentOS repositories
```

File: docs/alt-installation.md [hunk 11]
```diff
@@ -24,6 +24,91 @@ and then restart your shell.
...
+
+Centos users can use our rpm repo to install earthly.
```
Comment (dchw) [inline docs/alt-installation.md:95]:
```suggestion
CentOS users can use our rpm repo to install earthly.
```

Comment (dchw) [inline docs/alt-installation.md:95]:
```suggestion
CentOS users can use our rpm repo to install earthly.
```

File: docs/alt-installation.md [hunk 12]
```diff
@@ -24,6 +24,91 @@ and then restart your shell.
...
+
+1. Install utils required to manage yum repositories:
+
+    sudo yum install -y yum-utils
+
+2. Add the earthly repo to your system:
```
Comment (dchw) [inline docs/alt-installation.md:101]:
```suggestion
2. Add the Earthly repo to your system:
```

File: docs/alt-installation.md [hunk 13]
```diff
@@ -24,6 +24,91 @@ and then restart your shell.
...
+
+    sudo yum-config-manager \
+        --add-repo \
+        https://pkg.earthly.dev/earthly.repo
+
+3. Install earthly:
```
Comment (dchw) [inline docs/alt-installation.md:107]:
```suggestion
3. Install Earthly:
```

**Review by dchw — APPROVED** [review]
Some capitalization

File: docs/alt-installation.md (hunk 2 above)
Comment (alexcb) [reply docs/alt-installation.md:33]: the leading spaces will turn this into a codeblock.

**Comment by alexcb** [comment]
> Some capitalization

Thanks! I preemptively opened the PR without using the draft feature.

**Pull request merged**